libini2.c

```diff
@@ -202,70 +104,10 @@ char * read_token_from_ini(const char *ini_file,
 	return dup;
 }
 
-static void write_to_ini(struct load_store_params *params, const char *dev_name,
-		size_t name_len, const char *attr, const char *val, size_t len)
-{
-	FILE *f = params->f;
-
-	if (params->is_debug)
-		fwrite("debug.", 1, sizeof("debug.") - 1, f);
-	fwrite(dev_name, 1, name_len, f);
-	fwrite(".", 1, 1, f);
-	fwrite(attr, 1, strlen(attr), f);
-	fwrite(" = ", 1, sizeof(" = ") - 1, f);
-	fwrite(val, 1, len - 1, f);
-	fwrite("\n", 1, 1, f);
-}
-
-static int save_to_ini_dev_cb(struct iio_device *dev,
-		const char *attr, const char *val, size_t len, void *d)
-{
-	struct load_store_params *params = (struct load_store_params *) d;
-	const char *dev_name = iio_device_get_name(dev);
-	size_t name_len = dev_name ? strlen(dev_name) : 0;
-
-	if (attr_in_whitelist(attr, dev_name, name_len, params->is_debug,
-				params->whitelist, params->list_len))
-		write_to_ini(params, dev_name, name_len, attr, val, len);
-	return 0;
-}
-
-static int save_to_ini_chn_cb(struct iio_channel *chn,
-		const char *attr, const char *val, size_t len, void *d)
-{
-	struct load_store_params *params = (struct load_store_params *) d;
-	const char *dev_name = iio_device_get_name(params->dev);
-	size_t name_len = dev_name ? strlen(dev_name) : 0;
-	const struct iio_attr *attrib;
-	attrib = iio_channel_find_attr(chn, attr);
-	attr = iio_attr_get_filename(attrib);
-	if (attr_in_whitelist(attr, dev_name, name_len, false,
-				params->whitelist, params->list_len))
-		write_to_ini(params, dev_name, name_len, attr, val, len);
-	return 0;
-}
-
 void save_to_ini(FILE *f, const char *driver_name, struct iio_device *dev,
 		const char * const *whitelist, size_t list_len)
 {
-	unsigned int i;
-	struct load_store_params params = {
-		.dev = dev,
-		.whitelist = whitelist,
-		.list_len = list_len,
-		.is_debug = false,
-		.f = f,
-	};
-
 	write_driver_name_to_ini(f, driver_name);
-
-	/*for (i = 0; i < iio_device_get_channels_count(dev); i++)
-		iio_channel_attr_read_all(iio_device_get_channel(dev, i),
-				save_to_ini_chn_cb, &params);
-	iio_device_attr_read_all(dev, save_to_ini_dev_cb, &params);
-
-	params.is_debug = true;
-	iio_device_debug_attr_read_all(dev, save_to_ini_dev_cb, &params);*/
```

Comment:
@cristina-suteu you disabled this code in this commit (https://github.com/analogdevicesinc/iio-oscilloscope/commit/1a6775078d2038f35398a0dbacd47e6de9c7e6be). Is it not required at all?

Reply:
Oh, yes... it might be that I just blindly removed stuff without giving it a thought if it needs rewriting...

Reply:
Hmm, and I just see that likely all of the unused stuff is because of that commented code.... If it needs a rewrite, I can drop this commit

Reply:
I think it's best to drop the commit now. Keep it local or on a separate branch in case it turns out it is needed.